app/helpers/host_helper/textual_summary.rb

```diff
@@ -384,9 +404,9 @@ def textual_compliance_history
   def textual_users
     return nil if @record.is_vmware_esxi?
     num = @record.number_of(:users)
-    h = {:label => "Users", :image => "user", :value => num}
+    h = {:label => _("Users"), :image => "user", :value => num}
     if num > 0
-      h[:title] = "Show the #{pluralize(num, 'user')} defined on this VM"
+      h[:title] = _("Show the Users defined on this VM")
```

Comment:
This should correctly be:

`h[:title] = n_("Show the User defined on this VM", "Show the User defined on this VM", num)`

to reflect both singular / plural cases.